review.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import argparse, datetime, json, os, subprocess, requests
+
+
+def is_third_monday(d):
+    return d.weekday() == 0 and 15 <= d.day <= 21
+
```

Comment:
Nit: another newline here? (The amount of space between things is inconsistent in other places too.)